.idea/workspace.xml

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Drop this. Should probably update your local .gitignore to avoid this file in the future =D